src/tests/test_affiliate/WidgetsTest.php

```diff
@@ -19,23 +19,134 @@ class WidgetsTest extends TestCase {
      * Note that these tests will fail for non-Gutenberg WordPress (< version 5).
      */
 
+    // Keep track of posts created during the tests so that we can delete them on tear down.
+    static $postIDsToDelete = [];
+
+    static function tearDownAfterClass(): void
```

Comment:
See https://docs.phpunit.de/en/10.0/fixtures.html for documentation on PHPUnit's `tearDownAfterClass`.